testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/AbstractTestHiveViews.java

```diff
@@ -87,6 +87,31 @@ public void testArrayIndexingInView()
                 queryAssert -> queryAssert.containsOnly(row("hive")));
     }
 
+    @Test(groups = HIVE_VIEWS)
+    public void testWithClause()
```

Comment:
nit: `testCommonTableExpression` (CTE) since that's what most docs refer to this as?